src/main/java/mate/academy/Main.java

```diff
@@ -10,15 +10,15 @@
 import org.apache.logging.log4j.Logger;
 
 public class Main {
-    public static final Logger logger = LogManager.getLogger(OrderServiceImpl.class);
+    public static final Logger logger = LogManager.getLogger(Main.class);
 
     public static void main(String[] args) {
         AuthenticationService authenticationService = new AuthenticationServiceImpl();
         User user;
         try {
             user = authenticationService.login("bob", "1234");
         } catch (AuthenticationException e) {
-            logger.error("the user could not log in", e);
+            logger.error("the user could not log in");
             e.printStackTrace();
```

Comment:
again, why both logger.error and printStackTrace?)
does it make sense?

Reply:
sorry, I experimented and as a result did not change